shell/cmd.go

```diff
@@ -78,7 +78,7 @@ func RunNohupListener(displayServer string) {
 	switch displayServer {
 	case "wayland":
 		// run optimized wl-clipboard listener
-		utils.HandleError(nohupCmdWL("image").Start())
+		utils.HandleError(nohupCmdWL("image/png").Start())
```

Comment:
You should add a comment informing what  currently not supported. So apparently, jpeg and gif

Reply:
@ccoVeille  The behaviour for this is odd, using `image/png` does work for encoding jpeg data also, but I agree it would be a good idea to make this clear with a note.